admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/entity/Gender.java

```diff
@@ -0,0 +1,49 @@
+package io.mosip.kernel.syncdata.entity;
+
+import java.io.Serializable;
+
+import javax.persistence.AttributeOverride;
+import javax.persistence.AttributeOverrides;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+import io.mosip.kernel.syncdata.entity.id.CodeAndLanguageCodeID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+/**
+ * Gender entity mapped according to DB
+ * 
+ * @author Urvil Joshi
+ * @author Sidhant Agarwal
+ * @since 1.0.0
```

Comment:
Kindly add a note here that this is deprecated, and its added only for backward compatibility

Reply:
Added.